common/src/sql/dbinit.sql

```diff
@@ -264,13 +264,12 @@ CREATE TABLE omicron.public.silo_user (
     time_deleted TIMESTAMPTZ,
 
     silo_id UUID NOT NULL,
-    external_id TEXT
+    external_id TEXT NOT NULL
```

Comment:
I think this is missing `NOT NULL` on both `silo_id` and `external_id`.  But I'm not sure GitHub is showing me the right thing here.  Here's what it looks like:
![Screen Shot 2022-06-27 at 12 25 47 PM](https://user-images.githubusercontent.com/532688/176020119-9f55bc0d-037b-4b25-929d-bdc1801c1b9a.png)

which can't be right because L265 is different but isn't shown as a difference.

Reply:
my clone has

```sql
CREATE TABLE omicron.public.silo_user (
    id UUID PRIMARY KEY,
    time_created TIMESTAMPTZ NOT NULL,
    time_modified TIMESTAMPTZ NOT NULL,
    time_deleted TIMESTAMPTZ,

    silo_id UUID NOT NULL,
    external_id TEXT NOT NULL
);
```

this is weird